donut/modules/feedback/templates/feedback.html

```diff
@@ -28,27 +31,38 @@
                 </tr>
                 <tr>
                   <div class="form-group">
-                    <td class="feedback">Subject*: </td>
+                    <td class="feedback required">Subject: </td>
```

Comment:
Just make these `<label>`s since we already have these styles for `label.required`. Would also mean that clicking on them would focus on the input.

Reply:
That was the first thing I tried, but since it needs to be in a <td>, neither wrapping it internally nor externally in a label was successful. It's not great to have to duplicate a bit of formatting, but that's been done in a variety of places already.